### ISSUE TITLE
[Feature] Add Tutorials

### ISSUE THREAD

**Issue by Glitchy-Tozier** [8VIM/8VIM#82]
Hi!
@flide was nice enough to tell me how the 'selection keypad' works.

> When you swipe from right to center and hold, the gesture effectively starts selecting/highlighting text backward from the curson position. As you let go of the hold, the selection keypad opens up to fine tune your selection and allow for basic things with the selection like cut, copy, paste etc.

However, I still have no idea what that **o** in the middle of the arrows does.
I don't know what the key adjacent to the most bottom-left key does.
I have no clue what the white keys to.
And I'm certain there are funtionalities in the main keyboard that I'm unaware of.

This is why we need to add tutorials. Videos and interactive tutorials would be amazing, but for now, we can live with a few paragraphs, explaining how everything works.
(And seeing as how there is A LOT to talk about already and a lot of thought has gone into this keyboard, it is important to structure that text well.)

**Comment by Glitchy-Tozier** [comment]
Ok, I think I found out, what that bottom button does. It's a "select all" button I think. Still, my point stands.

Also there's white / invisible keys on the 'selection keypad', which _kind of_ act like a delete-key, but not quite.

**Comment by locnide** [comment]
I wonder if 8pen had tutorials to help you learn faster to type without looking like klavaro on desktop. I found mentions of "level of games" on reddit.

**Comment by sslater11** [comment]
For those new and wanting to practice, I'll share how I learn a new keyboard layout, which I seem to do a lot lmao.

It's very simple, and you can learn to touch type or learn a new android layout within a day or two, although obviously with a slow typing speed, but you'll know where all the keys are :D. You need to repeat it every morning for about a week to get faster.

1. Spend about 10 minutes typing out the alphabet in 3 letter groups.
I start by typing abc over and over until it feels like I've gained some speed.
I then type def, then ghi, etc.

So my practice can look like this, and sometimes even longer and can take up a whole android screen with it.
abc abc abc abc def def def def ghi ghi ghi ghi ghi ghi jkl jkl jkl jkl mno mno mno pqrst pqrst pqrst pqrst uvw uvw uvw uvw xyz xyz xyz xyz.

2. After that, I do it again to make sure I've not forgotten abc by this point.

3. Then I type out the whole alphabet a few times.

4. Just practice typing basic words like "hello, this is a just me testing if I can type properly". If you misspell a word, just type it a few times correctly so "hello this is kust lust just just just a test"

If this sounds boring, just pop a good song on(like Leadboots on Deezer - shameless plug lol) and that'll make it more fun.

**Comment by locnide** [comment]
I just though that it could be nice to have an option to not show the letters on the "wheel". This feature associated with a "klavaro-touch" software could be a nice way to be forced to learn the layout by heart.
*I think that not being able to see the keyboard layout really helped me to learn to type with all fingers on a desktop keyboard*
**edit**: moved to a new issue :  #146 (following flide's advice)

**Comment by flide** [comment]
@koalp We might add that option some day, not a priority currently. 
Also, not the correct thread to raise the issue. Raise a new issue for 'no letters on the wheel'.

**Comment by flide** [comment]
Maybe we should start working on the wiki on this repo to get these tutorials going.

**Comment by nerd190** [comment]
Hi, I too would appreciate this documentation, possibly some .gifs too and a youtube/asciicinema video made by one of yourselves. I use Neovim in Termux when away from the PC, I too hate how slow it is without an external keyboard, I have made custom layouts using "Jbak2Keyboard" & "Multiling O Keyboard" which made things easier, but this keyboard looks amazing!!! ...problem is, I cant work it out at all :'(
I sideloaded the tutorial app as its not available via the playstore anymore (perhaps adding the a link to a safe copy of the apk in the readme would be good, for ease and security), also the app hasnt been updated for 5 years now and I cant even view level 5, it opens but a blank screen, I tested this one 3 devices too (Samsung S5, android 8, Lineage OS. OnePlus 3T, android 9, Lineage OS. OnePlus 5T, android 10, Oxygen OS) all have the same result, cant view level 5. This means I can only learn how to type "to, too, at, tea" so I am completely lost, I cant even spell "hello", I am very keen/willing to learn but have no means to do so (no documentation or examples anywhere), but I can tell by your new layout & the old youtube clip that this is definately for me!!
Thank you, I look forward to the day I can use this!

**Comment by philippludwig** [comment]
> For those new and wanting to practice, 
> 1. Spend about 10 minutes typing out the alphabet in 3 letter groups.

@sslater11  This is not very helpful at all, because I can’t even manage to type an `a`.

**Comment by sslater11** [comment]
@philippludwig I was out on my deliveries and that made me burst out laughing haha.
Yeah it's a weird keyboard for sure!
I see that the link to the tutorial in the 8vim readme is dead.
Here's another video that shows how 8pen(which this keyboard this one is based on) works.
[https://www.youtube.com/watch?v=q3OuCR0EpGo](https://www.youtube.com/watch?v=q3OuCR0EpGo)

**Comment by philippludwig** [comment]
@sslater11 I watched that video, but I just can’t get 8vim to produce any letter at all. The video does not really apply, when I tried some of the motions shown in the video, nothing happens most of the time; once I managed to get a random emoji, but this was just by accident and I could not reproduce it.

If there was at least some instructions on how to produce any letter at all, maybe I could get the hang of it.
